docs/subscriptions/update.md

```diff
@@ -0,0 +1,162 @@
+# Update a subscription
+
+```text
+PATCH /subscriptions/{guid}
+```
+
+This endpoint allows clients to update information about a subscription. It returns the following information:
+
+:::{list-table}
+:header-rows: 1
+
+* - Field
+   - Type
+   - Required?
+   - Description
+* - `new_feed_url`
+   - String
+   - No
+   - The URL of the podcast RSS feed. Only returned if the `feed_url` field was updated by the request
+* - `is_subscribed`
+   - Boolean
+   - No
+   - Whether the user is subscribed to the podcast or not. Only returned if the `is_subscribed` field was updated by the request
+* - `subscription_changed`
+   - Datetime
+   - No
+   - The date on which the `is_subscribed` field was last updated. Presented in [ISO 8601 format](https://www.iso.org/iso-8601-date-and-time-format.html). Only returned if the `is_subscribed` field was updated by the request
+* - `guid_changed`
+   - Datetime
+   - No
+   - The date on which the podcast's GUID was last updated. Presented in [ISO 8601 format](https://www.iso.org/iso-8601-date-and-time-format.html). Only returned if the `guid` field was updated by the request
+* - `new_guid`
+   - String<UUID>
+   - No
+   - The new GUID associated with the podcast. Only returned if the `guid` field was updated by the request
+:::
+
+## Parameters
+
+The client must pass the subscription GUID in the query path and add at least one field update in the request body.
```

Comment:
Should we describe that sever & client projects can agree on additional field updates in the call, not described here, and that if the server doesn't recognise it, these should be ignored? For AntennaPod, for example, I can imagine we'll support calls to update all podcast-specific settings (like playback speed) in close collaboration with the Nextcloud plugin author, without (all of) these ending up in the specs.

In any case, if we would specify them all in these specs, we can't assume that all servers support it, because they simply might not have underlying functionality.

Reply:
@keunes would this kind of setting be handled by this endpoint? Or would that be considered an overload? Maybe this kind of granular control should be handled by a different endpoint which is more extensible and less core.

Reply:
@keunes I think having something similar to Gpodder's [settings endpoint](https://gpoddernet.readthedocs.io/en/latest/api/reference/settings.html) might be nice here.